src/libcompiler_builtins/lib.rs

```diff
@@ -180,7 +180,7 @@ pub mod reimpls {
             sr = sr.wrapping_add(1);
 
             // 1 <= sr <= u64::bits() - 1
```

Comment:
@nikomatsakis The comment is about the range of `sr`, not the amount of shift, so it is still correct.